<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormArticulo</class>
 <widget class="QWidget" name="FormArticulo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>577</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Cambria</family>
    <pointsize>12</pointsize>
    <italic>true</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_6">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>Icono</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QLabel" name="label_9">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>Articulo</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="LabelImage">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>Imagen</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_imagen_familia">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>Familia</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_imagen_modelo">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>Modelo</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="indent">
        <number>-1</number>
       </property>
       <property name="buddy">
        <cstring></cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLabel" name="label_imagen_moneda">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>Moneda</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QLabel" name="label_imagen_marca">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>Marca</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_imagen_unidad">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>Unidad</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::East</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="icon">
       <iconset resource="../Recursos.qrc">
        <normaloff>:/Imagenes/iconos/add.png</normaloff>:/Imagenes/iconos/add.png</iconset>
      </attribute>
      <attribute name="title">
       <string/>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="5" column="0">
            <widget class="QLabel" name="label_28">
             <property name="text">
              <string>Modelo:</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_29">
             <property name="text">
              <string>Marca:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1" colspan="2">
            <widget class="QComboBox" name="ComboModelo"/>
           </item>
           <item row="6" column="1" colspan="2">
            <widget class="QComboBox" name="ComboMarca"/>
           </item>
           <item row="8" column="1" colspan="2">
            <widget class="QComboBox" name="ComboMoneda"/>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_30">
             <property name="text">
              <string>Moneda:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Codigo:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Nombre:</string>
             </property>
            </widget>
           </item>
           <item row="15" column="2">
            <widget class="QToolButton" name="ButtonArchivoImagen">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="15" column="1">
            <widget class="QLineEdit" name="LineImagen">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="1" colspan="2">
            <widget class="QLineEdit" name="LineNombre"/>
           </item>
           <item row="15" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Imagen:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="2">
            <widget class="QLineEdit" name="LineCodigo">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QLineEdit" name="LineCodigoInterno">
             <property name="text">
              <string>0000</string>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="2">
            <widget class="QLineEdit" name="LineCodigoExterno">
             <property name="text">
              <string>0000</string>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Codigo Externo:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Codigo Interno:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1" colspan="2">
            <widget class="QComboBox" name="ComboTipo"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_27">
             <property name="text">
              <string>Familia:</string>
             </property>
            </widget>
           </item>
           <item row="13" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Cantidad Minima:</string>
             </property>
            </widget>
           </item>
           <item row="13" column="1" colspan="2">
            <widget class="QDoubleSpinBox" name="SpinCantidadMin">
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="maximum">
              <double>99999999.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="7" column="1" colspan="2">
            <widget class="QComboBox" name="ComboUnidad"/>
           </item>
           <item row="11" column="1" rowspan="2" colspan="2">
            <widget class="QComboBox" name="ComboVigencia">
             <item>
              <property name="text">
               <string/>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Vigente</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Caducado</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="11" column="0" rowspan="2">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Vigencia:</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_32">
             <property name="text">
              <string>Unidad:</string>
             </property>
            </widget>
           </item>
           <item row="9" column="0" rowspan="2">
            <widget class="QLabel" name="label_31">
             <property name="text">
              <string>Grupo:</string>
             </property>
            </widget>
           </item>
           <item row="9" column="1" rowspan="2" colspan="2">
            <widget class="QComboBox" name="ComboUnico">
             <item>
              <property name="text">
               <string/>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Unico</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>General</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="14" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Precio:</string>
             </property>
            </widget>
           </item>
           <item row="14" column="1" colspan="2">
            <widget class="QDoubleSpinBox" name="SpinPrecio">
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::NoButtons</enum>
             </property>
             <property name="maximum">
              <double>99999999.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="ButtonGuardar">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../Recursos.qrc">
               <normaloff>:/Imagenes/iconos/guardar.png</normaloff>:/Imagenes/iconos/guardar.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="ButtonModificar">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../Recursos.qrc">
               <normaloff>:/Imagenes/iconos/editar.png</normaloff>:/Imagenes/iconos/editar.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="ButtonEliminar">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../Recursos.qrc">
               <normaloff>:/Imagenes/iconos/delete.png</normaloff>:/Imagenes/iconos/delete.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="ButtonRegresar">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../Recursos.qrc">
               <normaloff>:/Imagenes/iconos/izquierda.png</normaloff>:/Imagenes/iconos/izquierda.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>ComboVigencia</tabstop>
  <tabstop>LineImagen</tabstop>
  <tabstop>LineNombre</tabstop>
  <tabstop>LineCodigo</tabstop>
  <tabstop>ButtonArchivoImagen</tabstop>
  <tabstop>LineCodigoInterno</tabstop>
  <tabstop>LineCodigoExterno</tabstop>
 </tabstops>
 <resources>
  <include location="../Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
